core/spec/models/spree/variant_property_rule_value_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Spree::VariantPropertyRuleValue, type: :model do
+  context "touching" do
+    let(:rule_value) { create(:variant_property_rule_value) }
+    let(:rule) { rule_value.variant_property_rule }
+
+    subject { rule_value.touch }
+
+    it "touches the variant property rule" do
+      expect { subject }.to change { rule.reload.updated_at }
```

Comment:
This will probably break on MySQL (second timestamp precision issue). Maybe we want to `rule.update_columns(updated_at: 1.day.ago)` before this line